### DISCUSSION TITLE
Update branch - enable 'No Volume' removes Mixer/Mixer Device settings

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1614] [Bug Reports]
@rern 

I applied the latest Update branch (March 2nd, 10:00 GMT) to my test rig to see what has been changing recently.  I understand that this is almost certainly work in progress but I though it would be useful to show you what I found. It appears in my setup that if the 'No Volume' setting is enabled then the Device / Mixer Device settings disappear - is this intended functionality?  No other settings where changed. Disabling 'No Volume' again, restores the Device / Mixer Device settings/ info.

With 'No Volume' disabled:
![rAudio_NoVolume_off](https://github.com/rern/rAudio/assets/25418025/03ad25e4-20dc-4a21-adea-f80fc1bad7bb)

With 'No Volume' enabled:
![rAudio_NoVolume_on](https://github.com/rern/rAudio/assets/25418025/b44e05dd-9fe3-4446-869c-84de76c577ba)

I hope this helps.

Chris

**Comment by rern** [comment]
Hi, Chris,

Please try the UPDATE again.

**Comment by ceejayemm** [comment]
@rern 
Thanks fro the very quick response, updating to the latest Update branch seems to fix this problem.

Many thanks

Chris
